openvino_tensorflow/executable.cc

```diff
@@ -24,8 +25,12 @@ using namespace ngraph;
 namespace tensorflow {
 namespace openvino_tensorflow {
 
-Executable::Executable(shared_ptr<Function> func, string device)
-    : m_device{device}, m_trivial_fn{nullptr}, m_function(func) {
+Executable::Executable(shared_ptr<Function> func, string device,
+                       string device_type)
```

Comment:
can we use "precision" for e.g. FP16 instead of "device_type" as it is already known by now, and would look cleaner in implementation than using GPU_FP16 again.

Reply:
Makes sense do implement it that way. But this will require further changes than "executable.cc" and can impact multiple things. Need to look into it first.